traveler-ios-ui/TravelerKitUI/Views/LabeledAnnotationView.swift

```diff
@@ -8,7 +8,76 @@
 
 import Foundation
 import MapKit
+import TravelerKit
 
-class LabeledAnnotationView: MKPinAnnotationView {
+open class LabeledAnnotationView: MKAnnotationView {
+    public var pinHorizontalMargin: CGFloat = 6
+    public var pinVerticalMargin: CGFloat = 4
+    public var pinArrowHeight: CGFloat = 6
+    public var pinBorderWidth: CGFloat = 1.5
 
+    public var isChosen: Bool = false {
+        didSet {
+            guard let parkingSpot = annotation as? ParkingSpot else {
+                Log("MapAnnotation not a ParkingSpot", data: nil, level: .error)
+                return
+            }
+
+            image = createAnnotationPinImage(withPrice: parkingSpot.parkingItem.price.valueInBaseCurrency, isChosen: isChosen)
+        }
+    }
+
+    override public func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        if selected {
+            isChosen = true
+        }
+    }
+
+    open func createAnnotationPinImage(withPrice price: Double, isChosen: Bool) -> UIImage {
```

Comment:
Rename this attribute, the annotation should not be aware of price. It should only be aware of a value.